docs/installation-and-operations/system-requirements/README.md

```diff
@@ -53,15 +53,103 @@ See [here](../operation/control/#scaling-the-number-of-web-workers) how to scale
+
+The following environment variables are relevant for performance.. [For more information on applying scaling options, please see this document](../installation/operation/scaling/).
+
+- `OPENPROJECT_WEB_WORKERS`: Number of web workers handling HTTP requests. Note that in Kubernetes deployments, this value is applied using replicas of the services.
+- `OPENPROJECT_WEB_TIMEOUT`: Maximum request processing time in seconds.
+- `OPENPROJECT_WEB_WAIT__TIMEOUT`: Timeout for waiting requests in seconds.
+- `OPENPROJECT_WEB_MIN__THREADS`: Minimum number of threads per web worker.
+- `OPENPROJECT_WEB_MAX__THREADS`: Maximum number of threads per web worker.
+- `OPENPROJECT_GOOD__JOB__MAX_THREADS`: Maximum number of threads for background workers.
+
+
+
+## Example Configurations
+
+### **Small Instance (≤ 200 users, low concurrent activity)**
+
+- **Database**: 2 CPU / 4 GiB RAM
+
+- **CPU**: 2 CPU
+
+- **RAM**: 4 GB
+
+- **Web Workers**:  2 Workers, each with 4 threads
+
+- **Background Workers**: 1 multithreaded worker with 4GiB RAM (more RAM possibly required for larger exports)
+
+- **Disk Space**: 20 GB + additional disk space in case of internal attachment storage
+
+### **Medium Instance (~500 users, moderate concurrent activity)**
+
+- **Database**: 2-4 CPU / 8 GiB RAM
+- **CPU**: 4 CPU
+- **RAM**: 8 GB
+-  **Web Workers**: 4 Workers, each with 4-8 threads
+  •
+- **Background Workers**: 2 multithreaded workers with 4-6 GiB RAM
+- **Disk Space**: 50 GB + additional disk space in case of internal attachment storage
+
+### **Large Instance (~1500 users, medium to high concurrent activity)**
+
+- **Database**: 4-8 CPU / 16 GiB RAM
+- **CPU**: 8 CPU
+- **RAM**: 16-24 GB
+- **Web Workers**: 6-8 Workers, each with 8-32 threads
+- **Background Workers**: 4-8 multithreaded workers with 4-6GiB RAM, depending on workload
+- **Disk Space**: 100 GB + additional disk space in case of internal attachment storage
+
+### **Enterprise-scale multitenancy instance (~80K - 100K users, high concurrent activity)**
+
+- **Database**: Cluster of two 8 vCPU / 32 GiB RAM (e.g., AWS db.m7g.xlarge, Gravitron 3)
+- **Worker instances**: 2-4 instances of the following
+  - **CPU**: 8 CPU (e.g., AWS r7a.xlarge instances)
+  - **RAM**: 32GB
+
+- **Web Workers**: 8 - 12 Workers, each with 8-32 threads and 6GiB available RAM
+- **Background Workers**: 8 multithreaded workers with 4-6GiB RAM, depending on workload
+- **Disk Space**: 250 GB + additional disk space in case of internal attachment storage
+
+
+
+### Additional Scaling Recommendations
+
+- **Monitor Resource Usage**: [Use our health checks to monitor for resource allocation and queueing information](../operation/monitoring/#health-checks). Adjust CPU, RAM, and disk space as needed.
```

Comment:
Just our health checks won't be enough to properly monitor this. Perhaps we should add a general remark towards resource monitoring using tools such as Grafana or what ever comes with the used cloud platform (cloudwatch etc.).